docs/source/user-guide/how-to-analyze-classifiers-and-datasets.rst

```diff
@@ -0,0 +1,61 @@
+
+.. figure:: ../../images/user-guide/how-to-analyze-classifiers-and-datasets/sample-viewer-display-mode-overlay.png
+    :alt: Sample Viewer Display Mode
+    :align: center
+
+    Figure 5: The display mode of the sample viewer is set to overlay.
+
+The attribution view shows a heatmap in input space that highlights which pixels contributed positively or negatively to the classification result (the heatmap is a visual representation of the numerical attribution, the color map used to render the heatmaps can be selected in the toolbox at the top of the ViRelAy user interface). Positive attribution means that the image region contributed positively towards the classification result, whereas negative attribution means that the image region contributed negatively, i.e., it speaks against the class that was the classification result. The overlay mode superimposes the heatmap onto the input image, thus enabling us to directly see the underlying image features. When the attributions are fine and detailed it usually makes sense to directly view the heatmaps, because the image details can be seen in the heatmaps. But when the attributions are coarse it is harder to correlate the heatmap to the corresponding image regions, therefore, the overlay mode makes it easier to find the actual image features that were attributed. In this example, the attributions are rather coarse, therefore the overlay mode should be favored. Figure 6 shows the attribution heatmaps superimposed onto the input images.
+
+.. figure:: ../../images/user-guide/how-to-analyze-classifiers-and-datasets/input-images-with-attribution-overlay-of-outlier-cluster.png
+    :alt: Input Images of an Outlier Cluster in the Class Horse with Attribution Overlay
+    :align: center
+
+    Figure 6: The input images of the samples in an outlier cluster of the class horse with the attribution heatmap superimposed onto them (sample vier display mode overlay).
+
+This overlay proves the hypothesis, that the classifier uses the copyright notices at the bottom of the images as feature to make its classification decision, because the positive attribution (bright red and yellow regions) are on the image region that contains the copyright notice. In fact the classifier seems to almost exclusively rely on the copyright notice as there is very little to no positive attribution on the image regions that contain the horses. In some cases, the horses even have some negative attribution on them.
```

Comment:
```suggestion
This overlay provides strong evidence in favor of the hypothesis, that the classifier uses the copyright notices at the bottom of the images as feature to make its classification decision, because the positive attribution (bright red and yellow regions) are on the image region that contains the copyright notice. In fact the classifier seems to almost exclusively rely on the copyright notice as there is very little to no positive attribution on the image regions that contain the horses. In some cases, the horses even have some negative attribution on them.
```

Reply:
Accepted